apps/api/src/app/controllers/__tests__/resourceDescriptions.e2e.spec.ts

```diff
@@ -35,10 +36,14 @@ describe('GET /resources', () => {
         const body = result.body as AggregateInfo[];
 
         // TODO [optimization]: avoid loop within loop here
-        const isThereAnEntryForEveryResourceType = Object.values(ResourceType).every(
-            (resourceType) =>
+        const isThereAnEntryForEveryResourceType = Object.values(ResourceType)
+            .filter(
+                (resourceType): resourceType is ResourceTypesWhoseViewsAreSourcedFromSnapshots =>
+                    resourceType !== ResourceType.digitalText
```

Comment:
I did this using a new type instead of declaring the const `resourceTypesThatHaveStandaloneQueryTests` by mistake (didn't re-find the example we'd used) but it seems to work.

Reply:
I added a comment for clarity.